### PR TITLE
feat(metrics): Track discarded events

### PR DESCRIPTION
**Pull request by macqueen**

### CODE REVIEW

File: src/sentry/event_manager.py [hunk 1]
```diff
@@ -671,6 +671,12 @@ def save(self, project, raw=False):
                 project=project,
                 sender=EventManager,
             )
+
+            metrics.incr(
+                'event.discarded',
+                skip_internal=True,
+                tags={'organization_id': project.organization_id},
```
Comment (mattrobenolt) [inline src/sentry/event_manager.py]: @JTCunning Any preference on this tag name? Pretty sure this is what we use elsewhere. Or should we use `instance` instead? I still don't know when we should use which.

**Comment by macqueen** [comment]
talked to ted and i'm going to update this to have an additional tag that'll allow us to count the number that needed to be processed vs not

**Comment by JTCunning** [comment]
@mattrobenolt: We've traditionally only done "project_id" so if we think
the organization is a better object to track, that's the way to do it.

**Comment by mattrobenolt** [comment]
Yeah, I'm explicitly referring to using the tag name `organization_id` vs using the `instance` kwarg.

I decided that we should track this per org instead of per project. Since the volume would be so low per project, and org more relates to usage and quotas. It'd make more sense for us to see "which orgs are using this the most".

**Review by tkaemming — CHANGES_REQUESTED** [review]
> talked to ted and i'm going to update this to have an additional tag that'll allow us to count the number that needed to be processed vs not

Recording here for posterity that I was wrong and you can just use the platform instead.

**Review by tkaemming — APPROVED** [review]
🍰

**Pull request merged**